apps/web/src/common/careActivites.json

```diff
@@ -0,0 +1,174 @@
+{
```

Comment:
We can get this data from the API as the API is up & running!

Reply:
Removed this and getting from api.